setup.cfg

```diff
@@ -21,7 +21,7 @@ max-line-length = 88
 [tool:pytest]
 addopts = -rsxX --tb=native --showlocals
 norecursedirs = .git docs bin
-testpaths = tests/unittest/
```

Comment:
I don't understand the thinking behind this. Why not put all the tests in `tests/unittest/`? Why create a `tests/external/` and then shift some things around? The `tests/README.rst` suggests each directory is a different test system, but they all run when you do `make test`, so I'm not sure I understand the breakdown.

As a side note, one of the things I wanted to do was re-root all the tests to just `tests/` rather than `tests/unittest/`. I forget why I structured it originally the way it's structured.

Reply:
cool, i'll file a bug to make that change after this PR is done